templates/2_Pixie_Cluster_Pixels.ipynb

```diff
@@ -31,18 +31,46 @@
     "from ark.utils import data_utils, example_dataset, plot_utils\n",
```

Comment:
<p>I don't think the paths "/Volumes" and "/Users/{username}" are true if you're not on a mac/not on docker. I don't think you need this note here to avoid confusion.</p><p> </p><p>Also, the text "the path to all your imaging data..." is repeated twice. Can delete the second one.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/1063/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>